nucypher/blockchain/eth/actors.py

```diff
@@ -131,7 +131,7 @@ def token_balance(self) -> NU:
         return nu_balance
 
 
-class Deployer(NucypherTokenActor):
+class DeployerActor(NucypherTokenActor):
```

Comment:
It wants another name. Any ideas?

Reply:
Can it not stay `Deployer`?  

Who is an actor which deploys a token or contract?  `Bursar`?  `Secretary`?  `Executive`?

Reply:
@jMyles - This name clashes with the namespace in `nucypher.blockchain.eth.deployers.Deployer`.